roles/nimbus/tasks/setup.yaml

```diff
@@ -24,6 +24,19 @@
     - "{{ nimbus_validator_datadir }}/secrets"
   when: nimbus_validator_enabled
 
+- name: Checkpoint sync nimbus node
+  when: nimbus_checkpoint_sync_enabled
+  community.docker.docker_container:
+    name: "{{ nimbus_checkpoint_container_name }}"
+    image: "{{ nimbus_container_image }}"
+    state: started
```

Comment:
I added `auto_remove` as a config var both on nimbus as well as geth, mainly because this container log info is super valuable on early testnets for debugging (Esp on shadowforks). So while its useless in a main network, the only time you would use it is in a testnet, i.e, set to false as its potentially useful information.